### PR TITLE
Add test for IPv4/IPv6 tunnel creation

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: stepanb <stepanb@mellanox.com>

**What I did**
I added VS test for IPv4/IPv6 tunnel creation

**Why I did it**
There was no VS test for tunnel creation, so I added it

**How I verified it**
Run VS tests

### CODE REVIEW

File: tests/test_tunnel.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+        # check asic db table
+        tunnel_table = swsscommon.Table(asicdb, self.ASIC_TUNNEL_TABLE)
+
+        tunnels = tunnel_table.getKeys()
+        assert len(tunnels) == 1
+
+        tunnel_sai_obj = tunnels[0]
+
+        status, fvs = tunnel_table.get(tunnel_sai_obj)
```
Comment (lguohan) [inline tests/test_tunnel.py]: we probably need to assert status == true here

File: tests/test_tunnel.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+        expected_ecn_mode = self.ecn_modes_map[ecn_mode]
+        expected_dscp_mode = self.dscp_modes_map[dscp_mode]
+        expected_ttl_mode = self.ttl_modes_map[ttl_mode]
+
+        for field, value in fvs:
```
Comment (lguohan) [inline tests/test_tunnel.py]: we need to access the len(fvs) since some attribute might be missing.

File: tests/test_tunnel.py (hunk 1 above)
Comment (stepanblyschak) [reply tests/test_tunnel.py]: Done

File: tests/test_tunnel.py (hunk 2 above)
Comment (stepanblyschak) [reply tests/test_tunnel.py]: Added assert len(fvs) == expected number of fields

**Comment by lguohan** [comment]
Retest this please

**Pull request merged**